### ISSUE TITLE
 I stop iterating twice

### ISSUE THREAD

**Issue by a1361098690** [brendanoconnor913/SFFS#2]
I distinguish the training set from the test set based on your code requirements and the correspon
![1](https://user-images.githubusercontent.com/45624213/59345988-e96beb80-8d43-11e9-87b2-ba69cca2d628.jpg)
![2](https://user-images.githubusercontent.com/45624213/59346002-eec93600-8d43-11e9-9f2b-fa50e22b4d34.png)

ding tags. Why did I stop iterating twice, I have a total of 96 features.
